app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@
 
 # ApplicationController
 class ApplicationController < ActionController::Base
+
+	private
+
+  def after_sign_out_path_for(resource_or_scope)
+    new_user_session_path
+  end
+
```

Comment:
Do we really need this?
Can we implement the logic of the dashboard which we have discussed last week? i.e (For the root path if user login then will be redirected to the dashboard otherwise to the new sessions path)